keps/sig-node/277-ephemeral-containers/README.md

```diff
@@ -629,10 +635,7 @@ via this subresource. `EphemeralContainerStatuses` is updated in the same manner
 as everything else in `Pod.Status` via `/status`.
 
 `Pod.Spec.EphemeralContainers` may be updated via `/ephemeralcontainers` as per
-normal (using PUT, PATCH, etc) except that existing Ephemeral Containers may not
-be modified or deleted. Deleting Ephemeral Containers is not supported in the
```

Comment:
was the restriction to modify/delete existing ephemeral containers lifted?

Reply:
Deletion was proposed and accepted in #1690, but it hasn't been implemented yet. Happy to revisit.

#1690 doesn't explicitly take a stance on in-place modifications of the mutable `Container` fields, but I don't intend to support it. I've re-added the "may not be modified" section here.